services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/HankeYhteystietoEntity.kt

```diff
@@ -121,4 +121,6 @@ data class Yhteyshenkilo(
     val sukunimi: String,
     val email: String,
     val puhelinnumero: String,
-)
+) {
+    fun wholeName(): String = "$etunimi $sukunimi".trim()
```

Comment:
The English for "koko nimi" is "full name".

I think joining the strings is more descriptive than concatenating a space and then possibly trimming it.
```suggestion
    fun fullName(): String = listOf(etunimi, sukunimi).joinToString(" ")
```

Reply:
Oh yeah, the names are not nulls, they're empty strings. So blanks need to be filtered out.
```suggestion
    fun fullName(): String = listOf(etunimi, sukunimi).filter { it.isNotBlank() }.joinToString(" ")
```

Reply:
What if the following input is received: etunimi: "", sukunimi: "Foo". Wouldn't the suggested approach end up with " Foo".